data/fm-info.json

```
@@ -10,823 +10,847 @@
-              {"memberHandle":"okayko","memberId":605,"inducted":null,"directScore":700,"referrer":{"memberHandle":"lopegor","memberId":1369,"inducted":{"fmId":8,"inductedScoringPeriodId":5,"inductedDate":"2021-04-23","avatar":"https://github.com/Joystream/founding-members/blob/main/avatars/primary-avatar/39.png?raw=true"},"referralScore":[0,100,25,51,71,150,200,80,80,140]}},
-              {"memberHandle":"leet_joy","memberId":957,"inducted":null,"directScore":650,"referrer":null},
-              {"memberHandle":"lyazufey1812","memberId":1905,"inducted":null,"directScore":600,"referrer":null}
+              {"memberHandle":"andybut","memberId":1316,"inducted":{"fmId":9,"inductedScoringPeriodId":7,"inductedDate":"2021-06-07","avatar":"https://github.com/Joystream/founding-members/blob/main/avatars/primary-avatar/1.png?raw=true"},"directScore":3750,"referrer":null},
+              {"memberHandle":"drmarkovi","memberId":982,"inducted":null,"directScore":3675,"referrer":null},
+              {"memberHandle":"maxlevush","memberId":2130,"inducted":{"fmId":15,"inductedScoringPeriodId":10,"inductedDate":"21.07.21","avatar":"https://github.com/Joystream/founding-members/blob/main/avatars/primary-avatar/29.png?raw=true"},"directScore":3250,"referrer":null},
+              {"memberHandle":"igrex","memberId":1048,"inducted":{"fmId":14,"inductedScoringPeriodId":10,"inductedDate":"21.07.21","avatar":"https://github.com/Joystream/founding-members/blob/main/avatars/primary-avatar/49.png?raw=true"},"directScore":1850,"referrer":{"memberHandle":"leet_joy","memberId":957,"inducted":null,"referralScore":[0,35,98,130,80,130,150,150,150,335,370]}},
+              {"memberHandle":"dapplooker","memberId":2183,"inducted":{"fmId":13,"inductedScoringPeriodId":10,"inductedDate":"21.07.21","avatar":"https://github.com/Joystream/founding-members/blob/main/avatars/primary-avatar/44.png?raw=true"},"directScore":1000,"referrer":null},
+              {"memberHandle":"joystreamenthusiast","memberId":555,"inducted":{"fmId":12,"inductedScoringPeriodId":9,"inductedDate":"2021-07-05","avatar":"https://github.com/Joystream/founding-members/blob/main/avatars/primary-avatar/37.png?raw=true"},"directScore":700,"referrer":null},
+              {"memberHandle":"ururu","memberId":790,"inducted":null,"directScore":550,"referrer":null},
+              {"memberHandle":"0x2bc","memberId":2098,"inducted":null,"directScore":500,"referrer":null},
+              {"memberHandle":"botzmann","memberId":604,"inducted":null,"directScore":475,"referrer":null},
+              {"memberHandle":"olgakuzia","memberId":2345,"inducted":null,"directScore":475,"referrer":{"memberHandle":"razumv","memberId":1019,"inducted":null,"referralScore":[0,0,0,0,0,0,0,0,0,0,95]}}
           ]
         }
     },
     "scores":
     {
-        "cutoff": "2021-06-24T14:09:34.379Z",
+        "cutoff": "2021-07-13T14:42:56.253Z",
```

@bwhm Did notice something, I stopped after I got to 120 hours after the period ended, so the cutoff is actually 2021-07-11 at 00:00.
```suggestion
        "cutoff": "2021-07-11T00:00:00.000Z",
```

---

Not sure how to do midnight in this format, correct if wrong @bwhm!